sdk/physical/physical.go

```diff
@@ -185,37 +185,6 @@ type Lock interface {
 // Factory is the factory function to create a physical backend.
 type Factory func(config map[string]string, logger log.Logger) (Backend, error)
 
-// PermitPool is used to limit maximum outstanding requests
```

Comment:
Should this not be marked as deprecated and the existing `NewPermitPool` return the shared version for at least some backwards compatibility since this is the SDK package?

Touchy as the Acquire method has changed 🤔

Reply:
`existing NewPermitPool return the shared version` sorry late in the day comment. I meant wrap the new shared implementation within itself and delegate to it internally from the existing methods.

Reply:
Great question, I didn't clock that this was a package users may depend on. I'm sort of worried that adding a wrapper here will just make users in this repo keep using it, but I suppose that's a risk we have to take. I'll add a wrapper.

Reply:
I added a wrapper which should be code-compatible, please take a look